src/main/java/epicsquid/roots/entity/ritual/EntityRitualFlowerGrowth.java

```diff
@@ -20,21 +27,42 @@ public EntityRitualFlowerGrowth(World worldIn)
     public void onUpdate()
     {
         ticksExisted++;
```

Comment:
At least from what I understand of entities, the `ticksExisted` field is automatically updated by `World::updateEntities`. I noticed that you changed your 100 to a 200, implying it was happening too quickly? Probably because it was double-ticking. 

From `World.java:1862` (at least in my decompiled version)

```java
            try
            {
                if(entity.updateBlocked) continue;
                ++entity.ticksExisted;
                entity.onUpdate();
            }
```

Reply:
Well, I saw it was used in others ritual entities as well so I added it as well...
but I had no idea that `ticksExisted` gets automatically updated
https://github.com/EpicSquid/Roots/blob/master/src/main/java/epicsquid/roots/entity/ritual/EntityRitualLife.java#L26

Reply:
It has that in every ritual. Worth testing without to see if it is working now

Reply:
From what I can tell entity.ticksExisted is ticked before onUpdate is called unless it's blocked, at least by `updateEntityWithOptionalForce`: ![](https://i.imgur.com/Ol1tXGh.png)